app/models/landmark.rb

```diff
@@ -7,9 +7,25 @@ class Landmark < ActiveRecord::Base
   validates :image, presence: true
   validates :description, presence: true
   validates :user, presence: true
-
+  validates :average_review, inclusion: { in: %w(0 1 2 3 4 5) }
 
   def self.search(search)
     where("name ILIKE ?", "%#{search}%")
   end
+
+  def average_rating
+    reviews = self.reviews
```

Comment:
@mlg- also, additional thoughts on this (mostly for me when I look at this again later & it pertains to comments below): it won't work unless we switch back over to integers for the ratings